frontend/packages/ceph-storage-plugin/src/components/bucket-class/create-bc.tsx

```diff
@@ -138,6 +163,9 @@ const CreateBucketClass: React.FC<CreateBCProps> = ({ match }) => {
     if (state.namespacePolicyType === NamespacePolicyType.SINGLE) {
       return state.readNamespaceStore.length === 1 && state.writeNamespaceStore.length === 1;
     }
+    if (state.namespacePolicyType === NamespacePolicyType.MULTI) {
+      return state.readNamespaceStore.length >= 1 && state.writeNamespaceStore.length === 1;
```

Comment:
why its >=1 and for single equal to  1 L164 ?

Reply:
Multi namespace bucket class can have multiple read targets whereas single can have only one

Reply:
okay